independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartCatalog.java

```diff
@@ -38,6 +41,11 @@
     public static final String INPUT_SELECTED_EXTENSIONS_KEY = "selected-extensions";
     public static final String INPUT_SELECTED_EXTENSIONS_GA_KEY = "selected-extensions-ga";
     public static final String INPUT_PROVIDED_CODE_KEY = "provided-code";
+    private static final String IO_QUARKUS_GROUP_ID = "io.quarkus";
+    private static final String IO_QUARKUS_PLATFORM_GROUP_ID = "io.quarkus.platform";
```

Comment:
It could also be a productized platform, which will have a different groupId.

Reply:
I agree, but for 2.8/2.9 that would (most likely) not apply. To handle that it would be better checking platform property.

Reply:
Right. I suppose it will "break" when using dev tools 2.8 with the next rh release (post 2.7).